src/main/kotlin/org/rust/lang/core/psi/ext/RsElement.kt

```diff
@@ -145,3 +146,10 @@ fun RsElement.findInScope(name: String): PsiElement? {
     }
     return resolved
 }
+
+
+val RsElement.isLocal: Boolean
+    get() {
+        val context = PsiTreeUtil.getContextOfType(this, true, RsItemElement::class.java, RsFile::class.java)
+        return context != null && context !is RsMod
```

Comment:
Is it expected that context can be RsMembers and so methods will be classified as local?

Reply:
And enum item too

Reply:
> Is it expected that context can be trait or impl and so methods themselves will be classified as locals?

I think yes. Examples from Kotlin:
![image](https://user-images.githubusercontent.com/6079006/55755226-228f9300-5a57-11e9-8d13-465a28a5dab6.png)

![image](https://user-images.githubusercontent.com/6079006/55755481-c9742f00-5a57-11e9-8d09-add556d7ab0e.png)